master/internal/configpolicy/postgres_task_config_policy.go

```diff
@@ -113,3 +114,55 @@ func DeleteConfigPolicies(ctx context.Context,
 	}
 	return nil
 }
+
+// GetEnforcedConfig gets the fields of the global invariant config or constraint if specified, and
+// the workspace invariant config or constraint otherwise. If neither is specified, returns nil.
```

Comment:
Nice function description. I recommend describing what the function does, and then describing the priority order. So something like 
```
// GetEnforcedConfig fetches the field from an invariant_config or constraints policyType, 
in order of precedence. Global scope has highest precedence, then workspace. 
Returns nil if none is found.
```

Reply:
I find the field format to be unintuitive. e.g. `"'resources' -> 'max_slots'"`

Could you add an example to the function description? Or at least as a comment within the function?

Reply:
I like the func description change! Changed this and added an example for the `field` arg